ods_ci/tests/Tests/500__jupyterhub/elyra-pipelines.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Documentation    Test Suite for Elyra pipelines in the SDS image
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/Elyra.resource
+Resource         ../../Resources/Page/ODH/Pipelines/Pipelines.resource
+Resource         ../../Resources/Page/OCPDashboard/Builds/Builds.robot
+Library          Screenshot
+Library          DebugLibrary
+Library          JupyterLibrary
+Suite Teardown   End Web Test
+Force Tags       JupyterHub
+
+
+*** Variables ***
+${SVG_CANVAS} =    //*[name()="svg" and @class="svg-area"]
+${SVG_INTERACTABLE} =    /*[name()="g" and @class="d3-canvas-group"]
+${SVG_PIPELINE_NODES} =    /*[name()="g" and @class="d3-nodes-links-group"]
+${SVG_SINGLE_NODE} =    /*[name()="g" and contains(@class, "d3-draggable")]
+
+
+
+*** Test Cases ***
+Verify Pipeline Is Displayed Correctly
+    [Documentation]    Loads an example Elyra pipeline and confirms the Web UI displays it correctly
+    [Tags]    Sanity    Tier1
+    ...       ODS-2197
+    [Setup]    Elyra Pipelines SDS Setup
+    # Until https://github.com/elyra-ai/examples/pull/120 is merged
+    Clone Git Repository And Open    https://github.com/lugi0/examples    examples/pipelines/run-generic-pipelines-on-kubeflow-pipelines/hello-generic-world.pipeline
+    #Clone Git Repository And Open    https://github.com/elyra-ai/examples    examples/pipelines/run-generic-pipelines-on-kubeflow-pipelines/hello-generic-world.pipeline
+    Verify Hello World Pipeline Elements
+
+Verify Pipeline Can Be Submitted And Runs Correctly
```

Comment:
Consider renaming to `Verify Pipeline Can Be Submitted And Runs Correctly In Elyra`

Reply:
I think `In Elyra` is wrong in this case, as the pipeline will run in kubeflow

Reply:
My suggestion was to add Elyra somewhere in the title (another example: `Verify Pipeline Can Be Submitted Using Elyra And Runs Correctly`) but if you don't see it necessary its fine for me